ontobio/io/gafparser.py

```diff
@@ -418,11 +418,15 @@ def to_association(gaf_line: List[str], report=None, group="unknown", dataset="u
             conjunction = association.ExtensionConjunctions(extension_units)
             conjunctions.append(conjunction)
     object_extensions = association.ExtensionExpression(conjunctions)
+    looked_up_rel = relations.lookup_label(relation)
+    if looked_up_rel is None:
+        report.error(source_line, assocparser.Report.INVALID_QUALIFIER, relation, "Qualifer must be \"colocalizes_with\" or \"contributes_to\"", taxon=taxon, rule=1)
```

Comment:
Or "NOT"?

Reply:
I didn't include NOT since NOT is filtered out of the "relation" like qualifiers. And so NOT is not examined in the same way as the other qualifiers are here. If that is confusing though, I can say NOT also.